mods/d2k/rules/arrakis.yaml

```diff
@@ -32,11 +32,11 @@ sandworm:
 		Description: Attracted by vibrations in the sand.\nWill eat units whole and has a large appetite.
 	Health:
 		HP: 1200
```

Comment:
Partially related: it takes 2 trikes to kill a worm before it can eat one of them.
TibEd says Sandworm has 9990 HP, so this definitely needs a buff.